index.js

```diff
@@ -99,8 +98,10 @@ app.post('/', function(req, response) {
   			// answer fetched from susi
 			ans = (JSON.parse(body1)).answers[0].actions[0].expression;
   			
-			// setting options to request the chat api of viber.
-			var options =   {
+			// checking type of json response
+			if((JSON.parse(body1)).answers[0].actions.length == 3 &&  (JSON.parse(body1)).answers[0].actions[2].type == "map" ){
```

Comment:
We can create an issue for the same on the server instead of checking this on the client, I guess?